popde/utils_plot.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+
+def simple2Dplot(xx, yy, kde):
+    """
+    return kde plot in 2D given output
+    """
+    fig = plt.figure()
+    ax = fig.gca()
+    # Contourf plot
```

Comment:
The comment is not informative for someone who does not already know 'contourf'. Suggest 'Filled contour plot'